packages/react/src/FronteggProvider.tsx

```diff
@@ -102,6 +109,10 @@ export const FronteggProvider: FC<FronteggProviderProps> = (props) => {
     );
   }
 
+  if (props.navigate) {
```

Comment:
we already have some other hooks that uses the useHistory, I prefer to make custom provider that register out useHistory context and use it everywhere